scripts/release-test.sh

```
@@ -116,6 +116,108 @@ set -e
+  "$prerequisite_log_dir/00-prerequisites.log"
+
+tool_log="$TEST_ROOT/release-tools.log"
+printf '%s\n' \
+  '#!/usr/bin/env bash' \
+  'printf '\''security %s\n'\'' "$*" >> "$TOOL_LOG"' \
+  'if [[ "$*" == "find-identity -v -p codesigning" ]]; then' \
+  '  printf '\''1) ABC "Developer ID Application: Test (337D6CNU4E)"\n'\''' \
+  'fi' > "$fake_bin/security"
+printf '%s\n' \
+  '#!/usr/bin/env bash' \
+  'printf '\''xcrun %s\n'\'' "$*" >> "$TOOL_LOG"' > "$fake_bin/xcrun"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)